packages/web-runtime/src/components/TopBar.vue

```diff
@@ -1,33 +1,27 @@
 <template>
   <header
-    class="oc-topbar uk-flex uk-flex-middle uk-flex-wrap oc-border-b oc-p-s"
+    id="oc-topbar"
+    class="uk-flex uk-flex-middle uk-flex-between oc-px-m"
     :aria-label="$gettext('Top bar')"
   >
-    <oc-grid gutter="medium" flex>
-      <div class="uk-hidden@l">
-        <oc-button
-          appearance="raw"
-          class="oc-m-s oc-app-navigation-toggle"
-          :aria-label="$gettext('Open navigation menu')"
-          @click="toggleAppNavigationVisibility"
-        >
-          <oc-icon name="menu" />
-        </oc-button>
-      </div>
-    </oc-grid>
+    <div class="gap-30 uk-flex uk-flex-middle uk-flex-between">
+      <applications-menu v-if="applicationsList.length > 0" :applications-list="applicationsList" />
```

Comment:
explicit `> 0` needed?

Reply:
Nope, thx. The whole component will receive a proper refactoring soon, hopefully